src/libs/actions/Session/Reauthenticator.ts

```diff
@@ -0,0 +1,67 @@
+import Onyx from 'react-native-onyx';
+import {reauthenticate} from '@libs/Authentication';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type Session from '@src/types/onyx/Session';
+
+let isOffline = false;
+let active = false;
+let currentActiveSession: Session = {};
+let timer: NodeJS.Timeout;
+const TIMING_BEFORE_REAUTHENTICATION_MS = 8500; // 8.5s
```

Comment:
What's the reason behind this number?

Reply:
i added this comment in the code for that variable

> // the delay before requesting a reauthentication once activated
// we think in that timing a "natural" reauthentication could happen (a session expired in the carousel is the only exception)
// and we wish not to overlap and make a double reauthentication